data/blog/opentelemetry-docker.mdx

```diff
@@ -215,7 +222,11 @@ service:
   pipelines:
     metrics:
       receivers: [otlp, docker_stats]
-      processors: [resourcedetection, batch]
+      processors: [resourcedetection/docker, batch]
+      exporters: [otlp]
+    metrics/internal:
+      receivers: [prometheus, hostmetrics]
+      processors: [resourcedetection/env, resourcedetection/system,resourcedetection/docker, batch]
```

Comment:
These `prometheus` and `hostmetrics` receivers don't exist in this example config. You can remove the `metrics/internal` pipeline.